sandbox-backend/src/main/java/com/exadel/sandbox/team5/util/AmazonClientS3.java

```diff
@@ -0,0 +1,17 @@
+package com.exadel.sandbox.team5.util;
+
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import org.springframework.context.annotation.Bean;
+
+public class AmazonClientS3 {
+
+    @Bean
+    public static AmazonS3 createClient() {
```

Comment:
I think it is better to use add `@Lazy` annotation, ( `https://www.baeldung.com/spring-lazy-annotation`). It will not block launch of application if we don't have credentials.

Reply:
done